.github/workflows/repo-check.yml

```diff
@@ -106,6 +106,8 @@ jobs:
     - name: get doxygen and graphviz
       run: |
         sudo apt-get install doxygen graphviz
+        cd doc
+        doxygen -u CodeDocumentation.conf.in 2>/dev/null
```

Comment:
It is probably better to commit such an update to the repo using a newer version of doxygen. I'm not sure if that will break older doxygen versions.

Reply:
I assumed it will, but I could be wrong

Reply:
Addressed in [a44e275](https://github.com/mfem/mfem/pull/3162/commits/a44e27583c0cf09db5af96ffda9ec46c05562d19)

Reply:
You used very recent version of doxygen (1.9.4) to update the `.conf` file and that created warnings for the doxygen version in `ubuntu-latest` which is v1.8.17.

It looks like it is better to not update the `.conf` file in the repo and just run `doxygen -u ...` here as you had it configured.

Reply:
Reverted

Reply:
Minor suggestion: why redirect stderr to `/dev/null` here? It could be useful to see the full output of the command in the log files.

Reply:
I get this output, which looked like the same one from stderr:

```
> doxygen -u CodeDocumentation.conf.in
warning: Tag 'TCL_SUBST' at line 237 of file 'CodeDocumentation.conf.in' has become obsolete.
         This tag has been removed.
warning: Tag 'COLS_IN_ALPHA_INDEX' at line 1039 of file 'CodeDocumentation.conf.in' has become obsolete.
         This tag has been removed.
warning: Tag 'LATEX_SOURCE_CODE' at line 1740 of file 'CodeDocumentation.conf.in' has become obsolete.
         This tag has been removed.
warning: Tag 'RTF_SOURCE_CODE' at line 1814 of file 'CodeDocumentation.conf.in' has become obsolete.
         This tag has been removed.
warning: Tag 'DOCBOOK_PROGRAMLISTING' at line 1912 of file 'CodeDocumentation.conf.in' has become obsolete.
         This tag has been removed.
warning: Tag 'PERL_PATH' at line 2092 of file 'CodeDocumentation.conf.in' has become obsolete.
         This tag has been removed.
warning: Tag 'CLASS_DIAGRAMS' at line 2105 of file 'CodeDocumentation.conf.in' has become obsolete.
         This tag has been removed.
warning: Tag 'MSCGEN_PATH' at line 2114 of file 'CodeDocumentation.conf.in' has become obsolete.
         This tag has been removed.


Configuration file 'CodeDocumentation.conf.in' updated.
```

Reply:
Done in https://github.com/mfem/mfem/pull/3162/commits/0040a7bc0eaeee81cfe94883c5ce7bef69332371